src/program/state_transition.py

```diff
@@ -17,45 +17,62 @@ def process_event(emitted_by: Service, existing_item: MediaItem | None = None, c
     no_further_processing: ProcessedEvent = (None, [])
     items_to_submit = []
 
-#TODO - Reindex non-released badly indexed items here
+    # Skip processing if item is paused
+    if existing_item and existing_item.is_paused:
```

Comment:
This lines: 20-25 is the only state_transition modification needed, remove the rest.